src/gei/Commands/MigrateRepo/MigrateRepoCommandHandler.cs

```diff
@@ -172,6 +174,9 @@ public async Task Handle(MigrateRepoCommandArgs args)
             _log.LogError($"Migration Failed. Migration ID: {migrationId}");
             _warningsCountLogger.LogWarningsCount(warningsCount);
             _log.LogInformation($"Migration log available at {migrationLogUrl} or by running `gh {CliContext.RootCommand} download-logs --github-target-org {args.GithubTargetOrg} --target-repo {args.TargetRepo}`");
+            Console.WriteLine($"Migration ID: {migrationId}, Org ID: {githubOrgId}, Source ID: {migrationSourceId}");
+
+            Console.WriteLine($"Error during migration: {failureReason}");
```

Comment:
These should probably be _log.LogError (or at least _log.something so that we capture in the logs what the user sees)